firmware/hw_layer/ports/stm32/flash_int.c

```diff
@@ -82,7 +82,10 @@ static bool intFlashUnlock(void) {
 /**
  * @brief Lock the flash memory for write access.
  */
-#define intFlashLock() { FLASH_CR |= FLASH_CR_LOCK; }
+static void intFlashLock(void)
```

Comment:
why do we make this macro a function?

Reply:
To be inline with intFlashUnlock
Also call of current macro cause useless semicolon:
`intFlashLock();`
Extended to
`{ FLASH_CR |= FLASH_CR_LOCK; } ;`

Reply:
@rusefillc for simple macros like this there is no value keeping it a macro, the compiler will inline it since the body is small